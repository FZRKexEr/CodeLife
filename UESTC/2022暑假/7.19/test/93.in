10
ijzyglezjepetpuqdksmuqubilypxstwivwniesnqmxubislzqqdnhjttonnjbrmhbcdodfromtbrgqujqcmzevortdogcobizjm
3
uqomotkzqtuuavswyoayjqgmiwefmznqntxvotzdfgrqaidmzxrjwfztutpwrszhqbintigqnhqdbakkfdrhkyltsqnssuplwfbi
10
fovbnmsqnruspmvxcverxrrsxsnczmypzfeaodhenwnilitmhgobgclmlzuktwhphncghqgamxntnoauzxyjlpdoxglwridrptdi
9
rlwjnxdfenzebzbljhcvxczhqbczatkfmypsmruvawqmktenlsanpbxxypjkrdmtplwbrqwzognbjjmewmlfdzyhvqozlahdolfl
10
kcsldvouvyvdapkzmzhekevodpewxbmpvgstbjpcbwvivclzvsmftffgpjmfaeueyuvcycygvioerbpzwhchduahjvyhgjflrapg
10
iesvtjqsmlxaidpvxnjuxrsanwcjakrmqhhgochrafhmihfuqwnteeantyragxaricuxouyckfdbqbjqoauibycarbwjvuhmcewd
9
zunyozwsyjgsfcepdzmcqviwkjsfomzytzfgyxvjbcuufweocyzmazcmbarvapddhdrjkzneyidwiqbdcaoklprouyxxbmejabma
4
ikucnbszfuxesabfkbnqrusdsljonnujzgiqbqnnbgorfupouzfghkvvtwuebdidtgwnoagizegebzidetncmucmnirzzqjlcafw
9
hbwowsrjumpucfgqrxascunnbpptpaplmfcttphikzhxiapxgksrnbpgrkwqttuvzroylspldgqiirhsjjpvlepgbnnstzupgoip
8
rdiohlebrxwhgnigdzojqeenfdrxeatzbfmpveyephiylwlmqpxdbntpndxphscvlfnsaqlqukubzitcqlsjgrizpsbobwiypwrg
10
