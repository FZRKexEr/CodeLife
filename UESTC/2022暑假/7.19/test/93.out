603979768
1024
2048
1024
1024
2048
33554432
2048
4096
1024
